app/components/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { ButtonProps, TouchableOpacity } from 'react-native'
+import { tailwind } from '../tailwind'
+import { Text } from './Text'
+
+export type ButtonFillTypes = 'fill' | 'outline' | 'flat'
+export type ButtonColor = 'primary' | 'secondary'
+
+export function Button (props: React.PropsWithChildren<ButtonProps> & { color?: ButtonColor, fill?: ButtonFillTypes, label?: string }): JSX.Element {
```

Comment:
Since this is an atomic component. I think it might be best to separate out the interface, contrary to my previous comment. Since this entire file serves a single purpose.

```suggestion
interface ButtonProps:  React.PropsWithChildren<ButtonProps> {
  color?: 'primary' | 'secondary'
  fill?: 'fill' | 'outline' | 'flat'
  label?: string
}

export function Button (props: ButtonProps): JSX.Element {
```